pkg/controller/persistentvolume/persistentvolume_recycler_controller.go

```diff
@@ -123,10 +188,28 @@ func (recycler *PersistentVolumeRecycler) reclaimVolume(pv *api.PersistentVolume
 			glog.Errorf(errMsg)
 			return fmt.Errorf(errMsg)
 		}
+		return nil
+	} else {
```

Comment:
By drop `else`, I mean you can remove the else clause but keep the glog statement.  This can be flattened by un-indenting and then return nil.